### PR TITLE
Track clicks on buttons

### PR DESCRIPTION
**Pull request by expe**
Added extra parameter in `dispatchCustomEvent()` for namespace, I added it as the last parameter to keep the usage the same but can move it if it makes more sense.

### CODE REVIEW

File: src/js/share.js [hunk 1]
```diff
@@ -30,12 +30,13 @@ function Share(rootEl, config) {
...
 
 	/**
-	  * Helper function to dispatch oShare namespaced events
+	  * Helper function to dispatch namespaced events, namespace defaults to oShare
 	  *
 	  * @private
 	  */
-	function dispatchCustomEvent(name, data) {
-		oShare.rootEl.dispatchEvent(new CustomEvent('oShare.' + name, {
+	function dispatchCustomEvent(event, data, namespace) {
```
Comment (AlbertoElias) [inline src/js/share.js]: you could use es6 default parameters :)

File: src/js/share.js [hunk 2]
```diff
@@ -49,6 +50,12 @@ function Share(rootEl, config) {
 	function handleClick(ev) {
 		const actionEl = ev.target.closest('li.o-share__action');
 
+		dispatchCustomEvent('event', {
```
Comment (AlbertoElias) [inline src/js/share.js]: this should probably go in the if down below

**Comment by expe** [comment]
Added es6 default params and moved event inside if statement

**Comment by AlbertoElias** [comment]
Nice! Lgtm!

**Pull request merged**